src/setupWorker/start/createStart.ts

```diff
@@ -65,29 +64,25 @@ export const createStart = (context: SetupWorkerInternalContext) => {
       )
 
       if (!instance) {
-        return null
+        throw new Error('[MSW] Failed to find locate service worker instance')
```

Comment:
Should we make the `getWorkerInstance` throw exceptions when failing to locate/create an instance as well? I'm speaking about these cases: 

https://github.com/mswjs/msw/blob/e23dca895156e944b4eb80ce24662711592fb227/src/setupWorker/start/utils/getWorkerInstance.ts#L77-L82

https://github.com/mswjs/msw/blob/e23dca895156e944b4eb80ce24662711592fb227/src/setupWorker/start/utils/getWorkerInstance.ts#L87-L90

If we throw exceptiont in the mentioned cases, we will be able to get that exception as the first argument of the `until`'s return tuple:

```js
const [instanceError, instance ] = await until(() => getWorkerInsance(...))

if (instanceError) {
  throw instanceError
}
```

Note that `until` gracefully handles promise exceptions and will not propagate them to the parental scope automatically. Instead, it will return them as the first item in the tuple.

Reply:
agreed. patched in 70ab1a4.  i saw a few places where `until` was warranted, but ended up pruning two uses where, per the above, we just plain wanted to throw, rather than fail, detect, and rethrow

Reply:
@cdaringe, makes sense. Thank you for the follow up on this!